src/cljs/mmm/components/projects.cljs

```diff
@@ -151,6 +151,17 @@
                [:p.tech "Github Pages, npm"
                 [:br] "Art by Brian (Narwhals of Sound)."]]
               "500"
-              "400"]]
+              "400"]
+
+            [project
+             "COPA-SMS"
+             "img/gallery/copa.gif"
+             [:h2
+              [:p "Copa new-member onboarding service via SMS with automatic user exports and archiving features."]
+              [:a {:class "rabbit"
+                   :href "https://coloradopeoplesalliance.org/"} "coloradopeoplesalliance.org"]
```

Comment:
Maybe we shouldn't even include the website at first? It might be confusing since we didn't make the website